docs/docs/reference/migration.md

```diff
@@ -13,6 +13,11 @@ Use [`constellation config migrate`](./cli.md#constellation-config-migrate) to a
   2. Set `useManagedIdentityExtension` to `true`  and use the `userAssignedIdentity` from the Constellation config for the value of `userAssignedIdentityID`.
   3. Restart the CSI driver, cloud controller manager, cluster autoscaler, and Constellation operator pods.
 
+
+## Migrating from CLI versions before 2.10
+
+- AWS cluster upgrades require additional iam permissions for the newly introduced `aws-load-balancer-controller`. Please perform an upgrade through `iam upgrade apply` which will show the needed changes and ask to confirm them before applying.
```

Comment:
```suggestion
- AWS cluster upgrades require additional IAM permissions for the newly introduced `aws-load-balancer-controller`. Please upgrade your IAM roles using `iam upgrade apply`. This will show necessary changes and apply them, if desired.
```